packages/trader/src/exchange-adapters/ExchangeAdapterInterface.ts

```diff
@@ -0,0 +1,19 @@
+class ExchangeAdapterInterface {
```

Comment:
all exchange adapters need to implement this interface, just like with price feeds.

Reply:
ok cool! I've gotten this to work. I export the interface as:
```
export default interface ExchangeAdapterInterface {
```

Then import it and use it as:
```
class UniswapTrader implements InstanceType<typeof ExchangeAdapterInterface> {
```